pkg/client/client.go

```diff
@@ -244,6 +246,8 @@ func (client *Client) Run(ctx context.Context, logger *zap.Logger, notifyCh chan
 
 			if client.options.Insecure {
 				opts = append(opts, grpc.WithInsecure())
+			} else {
+				opts = append(opts, grpc.WithTransportCredentials(credentials.NewTLS(&tls.Config{})))
```

Comment:
Heh, I stumbled on this too before :)

On the topic of dial options: we could add `grpc.WithUserAgent` for better logs on the load balancer / ingress.

Reply:
yep, I feel we should allow custom grpc client options, but this is a small fix to get things out working